test/resources/plan.sqlplan

```diff
@@ -0,0 +1 @@
perator><Const ConstValue="(1)"></Const></ScalarOperator></Compare></ScalarOperator><ScalarOperator><Compare CompareOp="EQ"><ScalarOperator><Identifier><ColumnReference Database="[master]" Schema="[sys]" Table="[sysschobjs]" Alias="[o]" Column="type"></ColumnReference></Identifier></ScalarOperator><ScalarOperator><Const ConstValue="&apos;V&apos;"></Const></ScalarOperator></Compare></ScalarOperator></Logical></ScalarOperator></Predicate></IndexScan></RelOp></ComputeScalar></RelOp><Predicate><ScalarOperator ScalarString="has_access(&apos;AO&apos;,[master].[sys].[sysschobjs].[id] as [o].[id])=(1)"><Compare CompareOp="EQ"><ScalarOperator><Intrinsic FunctionName="has_access"><ScalarOperator><Const ConstValue="&apos;AO&apos;"></Const></ScalarOperator><ScalarOperator><Identifier><ColumnReference Database="[master]" Schema="[sys]" Table="[sysschobjs]" Alias="[o]" Column="id"></ColumnReference></Identifier></ScalarOperator><ScalarOperator><Const ConstValue=""></Const></ScalarOperator><ScalarOperator><Const ConstValue=""></Const></ScalarOperator></Intrinsic></ScalarOperator><ScalarOperator><Const ConstValue="(1)"></Const></ScalarOperator></Compare></ScalarOperator></Predicate></Filter></RelOp></Hash></RelOp></Hash></RelOp></ComputeScalar></RelOp><RelOp NodeId="9" PhysicalOp="Clustered Index Seek" LogicalOp="Clustered Index Seek" EstimateRows="1" EstimatedRowsRead="1" EstimateIO="0.003125" EstimateCPU="0.0001581" AvgRowSize="71" EstimatedTotalSubtreeCost="0.123782" TableCardinality="163" Parallel="0" EstimateRebinds="0" EstimateRewinds="762.17" EstimatedExecutionMode="Row"><OutputList><ColumnReference Database="[mssqlsystemresource]" Schema="[sys]" Table="[syspalnames]" Alias="[n]" Column="name"></ColumnReference></OutputList><IndexScan Ordered="1" ScanDirection="FORWARD" ForcedIndex="0" ForceSeek="0" ForceScan="0" NoExpandHint="0" Storage="RowStore"><DefinedValues><DefinedValue><ColumnReference Database="[mssqlsystemresource]" Schema="[sys]" Table="[syspalnames]" Alias="[n]" Column="name"></ColumnReference></DefinedValue></DefinedValues><Object Database="[mssqlsystemresource]" Schema="[sys]" Table="[syspalnames]" Index="[cl]" Alias="[n]" IndexKind="Clustered" Storage="RowStore"></Object><SeekPredicates><SeekPredicateNew><SeekKeys><Prefix ScanType="EQ"><RangeColumns><ColumnReference Database="[mssqlsystemresource]" Schema="[sys]" Table="[syspalnames]" Alias="[n]" Column="class"></ColumnReference><ColumnReference Database="[mssqlsystemresource]" Schema="[sys]" Table="[syspalnames]" Alias="[n]" Column="value"></ColumnReference></RangeColumns><RangeExpressions><ScalarOperator ScalarString="&apos;OBTY&apos;"><Const ConstValue="&apos;OBTY&apos;"></Const></ScalarOperator><ScalarOperator ScalarString="&apos;V&apos;"><Const ConstValue="&apos;V&apos;"></Const></ScalarOperator></RangeExpressions></Prefix></SeekKeys></SeekPredicateNew></SeekPredicates></IndexScan></RelOp></NestedLoops></RelOp></QueryPlan></StmtSimple></Statements></Batch></BatchSequence></ShowPlanXML>
```

Comment:
Can we format this XML into something human-readable before checking it in?  It'd make future diffs easier.